gpflow/utilities/utilities.py

```diff
@@ -230,6 +231,33 @@ def update_state(parameter_or_variable, path, state):
     return state
 
 
+if Version(tfp.__version__) > Version("0.11.0"):
+
+    def _clear_bijector_cache(bijector: tfp.bijectors.Bijector):
+        bijector._cache.clear()
+
+
+elif Version(tfp.__version__) >= Version("0.11.0"):
```

Comment:
It's a bit weird that you have overlapping conditions. Was that intentional?
```suggestion
elif Version(tfp.__version__) == Version("0.11.0"):
```

Reply:
Good catch, it should be `==` only